### ISSUE TITLE
Ignore Blank Lines Feature?

### ISSUE THREAD

**Issue by Aetylus** [tora-tool/tora#144]
In Toad for Oracle, there's an option to "Treat blank line as statement terminator" that, when disabled, will ignore blank lines in statements. With that option disabled, the following will execute as a single SQL query:

```sql
select *

from employee

where id = 1;
```

However, in Tora, it appears to function as if the setting is enabled, and I was unable to locate an option to change this in the Preferences menu. This results in an error when attempting to execute the query:

```
last SQL statement:
select *

ORA-00923: FROM keyword not found where expected
```

Is there an option that exists for this?

**Comment by ibre5041** [comment]
Click in bottom right corner and switch into Oracle fuzzy parser.
I should be able to split statements by semicolons.

**Comment by Aetylus** [comment]
Thanks, that appears to work!

**Comment by Aetylus** [comment]
Actually, is there a way to make the option set by default for the SQL Editor?

**Comment by dglasser** [comment]
> 
> 
> Click in bottom right corner and switch into Oracle fuzzy parser.
> I should be able to split statements by semicolons.

@ibre5041,

I just installed TOra 3.2.186-gc5cb5fe8 for windows. I don't see any option for "Oracle fuzzy parser" anywhere. Are you referring to the control on the right side of the statusbar that switches between "Oracle", "QsciSql" and "None"? If so, which option selects the Oracle fuzzy parser?

Thanks.

**Comment by Aetylus** [comment]
@dglasser That's correct, the Oracle option is the fuzzy parser and will allow for blank lines to not be read as statement ends.
